kong/plugins/request-transformer/migrations/postgres.lua

```diff
@@ -0,0 +1,64 @@
+return {
+  {
+    name = "2016-05-04-160000_req_trans_schema_changes",
+    up = function(_, _, factory)
+
+      local plugins, err = factory.plugins:find_all {name = "request-transformer"}
+      if err then
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        for _, action in ipairs {"remove", "add", "append", "replace"} do
```

Comment:
Investigated this and adding the `rename` property is indeed not necessary, since it was introduced *after* this migration anyways, so this can be merged as-is. Waiting for CI and pulling this in in a few minutes. Thanks for keeping 👁 on this @john-paul-keenan